controllers/mysqlserver_controller.go

```diff
@@ -24,7 +33,66 @@ func (r *MySQLServerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 }
 
 func (r *MySQLServerReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// In order to watch a secret associated with this server:
+	// * We cannot use "For" - this automatically reconciles the type in question, and since the type
+	//   is a secret we don't want that.
+	// * We cannot use "Owns" - this reconciles the resources owner.
+	// * We must use Watches
+
+	// TODO:
+	// In order to actually make a change to the MySQLServer password, you must issue an update to the server which
+	// includes "administratorLogin": "<user>", "administratorLoginPassword": "<password>",
```

Comment:
Is this TODO still to be done?

Reply:
Oh, nope. Good eye. This was a note to myself that you must have both. There's a bug (I emailed them) if you omit the username it just hangs forever.